ui/imports/shared/popups/send/views/TokenListView.qml

```diff
@@ -80,85 +80,69 @@ Item {
 
-                    onCurrentIndexChanged: {
-                        if (currentIndex >= 0) {
-                            root.browsingHoldingType = d.holdingTypes[currentIndex]
-                        }
-                    }
+            anchors.fill: parent
+            anchors.topMargin: root.onlyAssets ? 0 : 20
 
-                    Repeater {
-                        id: tabLabelsRepeater
-                        model: d.tabsModel
+            StatusTabBar {
+                visible: !root.onlyAssets
+                Layout.preferredHeight: 40
+                Layout.fillWidth: true
+                currentIndex: d.holdingTypes.indexOf(root.browsingHoldingType)
 
-                        StatusTabButton {
-                            text: modelData
-                            width: implicitWidth
-                        }
+                onCurrentIndexChanged: {
+                    if (currentIndex >= 0) {
+                        root.browsingHoldingType = d.holdingTypes[currentIndex]
                     }
                 }
 
-                StatusListView {
-                    id: tokenList
-
-                    width: parent.width
-                    height: tokenList.contentHeight
-
-                    header: root.browsingHoldingType === Constants.TokenType.ERC20 ? tokenHeader : collectibleHeader
-                    model: root.browsingHoldingType === Constants.TokenType.ERC20 ? root.assets : collectiblesModel
-                    delegate: root.browsingHoldingType === Constants.TokenType.ERC20 ? tokenDelegate : collectiblesDelegate
-                    section {
-                        property: "isCommunityAsset"
-                        delegate: Loader {
-                            width: ListView.view.width
-                            required property string section
-                            sourceComponent: root.browsingHoldingType === Constants.TokenType.ERC20 && section === "true" ? sectionDelegate : null
-                        }
-                    }
-                }
+                Repeater {
+                    id: tabLabelsRepeater
+                    model: d.tabsModel
 
-                Component {
-                    id: sectionDelegate
-                    AssetsSectionDelegate {
-                        width: parent.width
-                        onOpenInfoPopup: Global.openPopup(communityInfoPopupCmp)
+                    StatusTabButton {
+                        text: modelData
+                        width: implicitWidth
                     }
                 }
+            }
+
+            StatusListView {
+                id: tokenList
 
-                Component {
-                    id: communityInfoPopupCmp
-                    CommunityAssetsInfoPopup {}
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+
+                header: root.browsingHoldingType === Constants.TokenType.ERC20 ? tokenHeader : collectibleHeader
```

Comment:
nit: same check done several times, could be replaced by some `isBrowsingTypeERC20` property defined once